i18n/fr-FR.yml

```diff
@@ -4,10 +4,86 @@ _name: Unofficial French Translations!
 components:
   DefaultItinerary:
     clickDetails: Cliquez pour afficher les détails
-    departureArrivalTimes: "{startTime}—{endTime}"
     multiModeSummary: "{accessMode} + {transitMode}"
-    tripDurationFormatZeroHours: "{minutes, number} mn"
-    tripDurationFormat: "{hours, number} h, {minutes, number} mn"
+  ItinerarySummary:
+    fareCost: "{useMaxFare, select, 
+      true {{minTotalFare} - {maxTotalFare}}
+      other {{minTotalFare}}
+    }"
+  NarrativeItinerariesHeader:
+    numIssues: "{issueNum, number} problèmes"
+    resultText: "{pending, select,
+      true {Recherche de vos options en cours...}
+      other {
+        {itineraryNum, number} {itineraryNum, plural,
+          one {trajet trouvé}
+          other {trajets trouvés}
+        }
+      }
+    }"
+    selectArrivalTime: Heure d'arrivée
+    selectBest: Meilleure option
+    selectCost: Prix
+    selectDepartureTime: Heure de départ
+    selectDuration: Durée
+    selectWalkTime: Temps de marche
+    titleText: "{pending, select, 
+      true {Recherche de vos options en cours...}
+      other {
+        {itineraryNum, number} {itineraryNum, plural,
+          one {trajet}
+          other {trajets}}
+        {issueNum, plural,
+          =0 {trouvé}
+          one {(et {issueNum, number} problème) trouvé}
+          other {(and {issueNum,number} problèmes) trouvés}
```

Comment:
```suggestion
          other {(and {issueNum, number} problèmes) trouvés}
```

Reply:
addressed in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/ad8ea3326848a5ac077e01898d7f7657412fda84